lib/versions/index.js

```diff
@@ -0,0 +1,29 @@
+const request = require('request-promise-native');
```

Comment:
I've updated the description, but I'm not sure what you're saying by inaccessible. Both seem to work fine independently from each other.

Reply:
Ah sorry, I meant inaccessible in that it doesn't show up under `--help`.

```
$ NODE_ENV=localhost ./rdme.js

Usage: rdme <command> [arguments]

Interact with your ReadMe project
  $ rdme login     Login to a ReadMe project
  $ rdme swagger   Upload your swagger file to ReadMe
  $ rdme docs      Sync a folder of markdown files to your ReadMe project
  $ rdme index     List versions available in your project
  $ rdme docs:edit Edit a single file from your ReadMe project without saving locally
  $ rdme versions:create Create a new version for your project
  $ rdme versions:delete Delete a version associated with your ReadMe project
  $ rdme versions:update Update an existing version for your project
  $ rdme versions:versionId List versions available in your project

Other useful commands
  $ rdme open      Open your current ReadMe project in the browser
  $ rdme oas       OAS related tasks. See https://www.npmjs.com/package/oas
```

Running it by itself works for me just fine, though it could use some prettifying up.

```
$ NODE_ENV=localhost ./rdme.js versions --key=...
[ { _id: '...',
    createdAt: '2019-06-17T22:53:32.432Z',
    is_deprecated: false,
    is_hidden: false,
    is_beta: false,
    is_stable: true,
    codename: '',
    version: '1.0' } ]
```

Reply:
Ahh i see, thanks for catching that. Fixed :) In regards to the output, do you have any suggestions?

Reply:
I was thinking of using yargs for the CLI rewrite we talked about yesterday, so maybe we could incorporate https://github.com/yargs/cliui and build some boxes in the terminal and kind of replicate how we display these versions in the dashboard.

```
| version | codename | is deprecated | is hidden | is beta | is stable | created |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| 1.0 | | false | false | false | false | true | 2019-06-17T22:53:32.432Z |
```
(but in a fancier form?)

Reply:
Sounds interesting. @dok suggested some help updates as well. Potentially we bundle these into a QOL PR? I'm happy to start working on those after this.